test/integration/acceptance/custom/hello_policy/init_test.go

```diff
@@ -48,25 +48,17 @@ func skupperInitEdgeTestScenario(ctx *base.ClusterContext, prefix string, withPo
 			{Ctx: ctx, Commands: []cli.SkupperCommandTester{
 				// skupper init - edge mode, no console and unsecured
 				&cli.InitTester{
-					ConsoleAuth:           "unsecured",
-					ConsoleUser:           "admin",
-					ConsolePassword:       "admin",
-					Ingress:               "none",
-					RouterDebugMode:       "gdb",
-					RouterLogging:         "trace",
-					RouterMode:            "edge",
-					SiteName:              "private",
-					EnableConsole:         false,
-					EnableRouterConsole:   false,
-					RouterCPU:             "100m",
-					RouterMemory:          "32Mi",
-					ControllerCPU:         "50m",
-					ControllerMemory:      "16Mi",
-					RouterCPULimit:        "600m",
-					RouterMemoryLimit:     "500Mi",
-					ControllerCPULimit:    "600m",
-					ControllerMemoryLimit: "500Mi",
-					//ConsoleIngress:      "none",
+					ConsoleAuth:         "unsecured",
+					ConsoleUser:         "admin",
+					ConsolePassword:     "admin",
+					Ingress:             "none",
+					RouterDebugMode:     "gdb",
+					RouterLogging:       "trace",
+					RouterMode:          "edge",
+					SiteName:            "private",
+					EnableConsole:       false,
+					EnableRouterConsole: false,
+					// ConsoleIngress:      "none",
 				},
```

Comment:
I have kept the original configuration, as I want to keep as close as possible to the original hello world test.